fastlane/lanes/release.rb

```diff
@@ -255,6 +255,11 @@ def trigger_buildkite_release_build(branch:, beta:)
     environment: { BETA_RELEASE: beta },
     pipeline_file: 'release-build.yml'
   )
+
+  return unless is_ci
+
+  message = "This build triggered #{build_url} on <code>#{branch}</code>."
```

Comment:
`build_url` variable is not defined here. You need to update line 251 to `build_url = buildkite_trigger_build(…)` too for this to be defined 😉